src/client/java/minicraft/screen/Toast.java

```diff
@@ -0,0 +1,80 @@
+package minicraft.screen;
+
+import minicraft.gfx.Dimension;
+import minicraft.gfx.Insets;
+import minicraft.gfx.Screen;
+import minicraft.gfx.Sprite;
+
+import java.util.List;
+
+public abstract class Toast {
+	protected static final int SPACING = 2;
+	protected static final int ANIMATION_TIME = 12;
+
+	protected final int expireTime;
+
+	protected int tick = 0;
+	protected int animationTick = 0; // 0 to ANIMATION_TIME, ANIMATION_TIME to 0
+
+	protected static abstract class ToastFrame {
```

Comment:
Remove the ToastFrame class as it is a single-variable class with no functionality that cannot be done in the Toast class. There is no need to create classes just for the fun of it.